extension/screenshooter/src/main/java/org/arquillian/extension/recorder/screenshooter/browser/impl/BrowserScreenshooter.java

```diff
@@ -145,18 +146,18 @@ public Screenshot takeScreenshot(File screenshotToTake, ScreenshotType type) {
         return screenshoot;
     }
 
-    private WebDriver getTakingScreenshotsBrowser() {
-
-        GrapheneContext context = null;
+    private WebDriver getTakingScreenshotsBrowser(File screenshotToTake) {
+        WebDriver result = null;
 
         try {
-            context = GrapheneContext.getContextFor(Default.class);
+            GrapheneContext context = GrapheneContext.getContextFor(Default.class);
+            result = ((GrapheneProxyInstance) context.getWebDriver(TakesScreenshot.class)).unwrap();
```

Comment:
The problem isn't the Graphene context, but that there's no WebDriver available. This happens if you mix in-container arquillian tests and Graphene tests in one project and use the same arquillian.xml.
If the arquillian.xml contains the following:

```
   <extension qualifier="screenshooter">
     ...
   </extension>
```
then the Screenshooter extension kicks in and tries to create a screenshot if the in-container test fails. But context.getWebDriver fails with 
"There is no context available for qualifier org.jboss.arquillian.drone.api.annotation.Default. Available contexts are []"